.github/release.yml

```
@@ -0,0 +1,19 @@
+changelog:
+  exclude:
+    labels:
+      - ignore-for-release
+    authors:
+      - JohnnyQ5
```

See newest release.yml of [template](https://github.com/qbicsoftware/spring-boot-rest-service-template/blob/main/.github/release.yml)](github-actions)
```suggestion
      - JohnnyQ5
      - github-actions
```